### DISCUSSION TITLE
11.0.0 Preview 8

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#11293] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/11.0.0-preview8'>11.0.0 Preview 8</a>.</em>

API change log: https://github.com/AvaloniaUI/Avalonia/pull/10051/commits/9b3c3314454815434e595dfc5e5111eb16f258d7
No significant breaking changes were made.

**Comment by mythay** [comment]
板凳。
I'm the very first guy see this release. Yeah!
Hope the official release.

**Reply by timunie** [reply]
? can you please translate into English ?

**Reply by culushishui** [reply]
if translate english,the meaning cannot be fully mapped

**Reply by timunie** [reply]
in that case I hope it's positive ^^

**Reply by rabbitism** [reply]
@timunie It means: "Now I have a room super close this thread starter."

**Reply by timunie** [reply]
Thx  🌺

**Reply by kyuranger** [reply]
> ? can you please translate into English ?
He means he is the first responder on this thread

**Reply by mythay** [reply]
> @timunie It means: "Now I have a room super close this thread starter."

The perfect translation.

**Comment by IsaacMarovitz** [comment]
Any breaking changes?

**Reply by maxkatz6** [reply]
You can see from this API diff. Nothing significant was changed API wise.
https://github.com/AvaloniaUI/Avalonia/pull/10051/commits/9b3c3314454815434e595dfc5e5111eb16f258d7

**Comment by npolyak** [comment]
We are getting there!

**Comment by kyuranger** [comment]
The animation bug in preview 7 has been fixed. Get well!

**Comment by orrest** [comment]
So excited about the Avalonia!
